docs/services/gcp-services-guide.md

```diff
@@ -0,0 +1,536 @@
+  # They are created in the service producer network (a VPC network internal to Google) that is then connected (peered) to your VPC network.
+  privateNetwork: projects/$PROJECT_ID/global/networks/$NETWORK_NAME
+  providerRef:
+    name: gcp-provider
+    namespace: $INFRA_NAMESPACE
+  reclaimPolicy: Delete
+EOF
+
+kubectl apply -f gcp-mysql-standard.yaml
+```
+* You can verify creation with the following command and output:
+
+*Command*
+```bash
+kubectl get cloudsqlinstanceclass -n ${INFRA_NAMESPACE}
+```
+*Output*
+```bash
+NAME                   PROVIDER-REF   RECLAIM-POLICY   AGE
+standard-cloudsql      gcp-provider   Delete           11s
+```
+
+You are free to create more GCP `CloudsqlInstanceClass` instances to define more
+potential configurations. For instance, you may create `large-gcp-mysql` with
+field `storageGB: 100`.
+
+#### Namespaces
+Kubernetes namespaces allow for separation of environments within your cluster.
+You may choose to use namespaces to group resources by team, application, or any
+other logical distinction. For this demo, we will create a namespace called
+`app-project1-dev`, which we will use to group our Wordpress resources.
+
+* Create a `Namespace`:
+```bash
+kubectl create namespace app-project1-dev
+
+```
+
+#### Portable Resource Classes
+Portable resource classes are used to define a class of service in a single
+namespace for an abstract service type. We want to define our GCP
+`CloudsqlInstanceClass` as the standard MySQL class of service in the namespace 
+that our Wordpress resources will live in.
+
+* Define a `MySQLInstanceClass` in `mysql-standard.yaml` for namespace
+  `app-project1-dev`:
+```bash
+cat > mysql-standard.yaml <<EOF
+---
+apiVersion: database.crossplane.io/v1alpha1
+kind: MySQLInstanceClass
+metadata:
+  name: mysql-standard
+  namespace: app-project1-dev
+classRef:
+  kind: CloudsqlInstanceClass
+  apiVersion: database.gcp.crossplane.io/v1alpha2
+  name: standard-cloudsql
+  namespace: $INFRA_NAMESPACE
+EOF
+
+kubectl apply -f mysql-standard.yaml
+```
+
+* You can verify creation with the following command and output:
+
+*Command*
+```bash
+kubectl get mysqlinstanceclasses -n app-project1-dev
+```
+*Output*
+```bash
+NAME             AGE
+mysql-standard   27s
+```
+
+Once again, you are free to create more `MySQLInstanceClass` instances in this
+namespace to define more classes of service. For instance, if you created
+`mysql-gcp-large` above, you may want to create a `MySQLInstanceClass` named
+`mysql-large` that references it. You may also choose to create MySQL resource
+classes for other non-GCP providers, and reference them for a class of service
+in the `app-project1-dev` namespace.
+
+You may specify *one* instance of a portable class kind as *default* in each
+namespace. This means that the portable resource class instance will be applied
+to claims that do not directly reference a portable class. If we wanted to make
```

Comment:
I am not sure I would understand this as a new user. Maybe it'd make sense to tell the user to run the `label` command so that we can provision our stuff, and then link them to a document which explains the concepts in more dept?

Reply:
Added a link and explanation about portable & cloud-specific resource classes.